packages/types/src/primitives/calendar-event.ts

```diff
@@ -23,4 +24,15 @@ export interface CalendarEvent {
   _level?: number;
   _overlapCount?: number;
   _overlapGroup?: string[];
+
+  // Properties for scheduled events
+  _isScheduledEvent?: boolean;
+  _scheduledEventId?: string;
+  _attendeeStatus?: EventAttendeeStatus;
+  _isPending?: boolean;
+  _isTentative?: boolean;
+  _isCreator?: boolean;
+  _attendeeCount?: Readonly<
+    { total: number } & Record<EventAttendeeStatus, number>
```

Comment:
should exists outside of the calendar event interface, ideally you reference it from a middleman table like `event_attendees` / `event_participants`.